src/app/components/PublisherCard/index.tsx

```diff
@@ -1,73 +1,91 @@
 
 export type Props = {
   title: string;
-  description?: string;
   image: string;
+  description?: string;
+  lnAddress?: string;
   url?: string;
+  isCard?: boolean;
+  isSmall?: boolean;
   children?: React.ReactNode;
 };
 
 export default function PublisherCard({
   title,
-  description,
   image,
+  description,
   url,
+  lnAddress,
+  isCard = true,
+  isSmall = true,
   children,
 }: Props) {
-  const { data } = usePalette(image);
   return (
     <div
-      className="p-4 bg-gray-300"
-      style={{
-        backgroundColor: data.vibrant,
-        backgroundImage: `linear-gradient(${data.vibrant}, rgba(0, 0, 0, 0.15))`,
-      }}
+      className={
+        "flex items-center bg-white dark:bg-surface-02dp " +
+        (isCard ? "drop-shadow rounded-lg mt-4 mx-4 " : "") +
+        (isSmall ? "p-2" : "flex-col justify-center p-4")
+      }
     >
-      <div className="flex sm:justify-center items-center">
-        <img
-          className="mr-3 w-14 h-14 shrink-0 border-solid border-2 border-white object-cover rounded-full shadow-xl"
-          src={image}
-          alt=""
-          onError={(e) => {
-            const target = e.target as HTMLImageElement;
-            target.onerror = null;
-            target.src = DEFAULT_IMAGE;
-          }}
-        />
-        <div className="overflow-hidden">
-          <h2
-            className="text-lg leading-5 font-bold text-white truncate"
-            title={title}
+      <img
+        className={
+          "m-2 shrink-0 bg-white border-solid border-2 border-white object-cover rounded-lg shadow-2xl " +
```

Comment:
For these smaller optional classes you can use template literals like i.e. here:
https://github.com/getAlby/lightning-browser-extension/blob/feat/1216_refactor-prompt-popup-lnurl-handling/src/app/components/Badge/index.tsx#L11